stdlib/docs/internal/README.md

```diff
@@ -0,0 +1,16 @@
+## WARNING
```

Comment:
@owenhilyard, not sure if "internal" is the best name for me it could be consider as a barrier to potential contributors. Even the plural form "internals" is clearer to me that we are documenting things happening underneath.

Anyways, I'm not a native English speaker, so maybe I'm misleading here.

Reply:
Good point. I'm trying to aggressively ward people away from using this unless they have no other choice since we want to keep the MLIR APIs and not technically stable parts of the language flexible, and I think that `internals` may not express what I want. Can I get some general feedback from readers on which of the following directory names seems like it wouldn't stop you from reading it as a prospective stdlib contributor but would stop you from using it at work (or other ideas)?

* internal
* internals
* stdlib_contributor
* unstable
* stdlib_only
* language_internals

Reply:
I don't feel strongly.  Any of them are fine and we can always easily rename the directory.